dashboard/app/lib/stylesheets/dashboard/items-list.scss

```diff
@@ -26,6 +26,9 @@
       background-color: $greenColor;
       color: $whiteColor;
     }
+
+    &.system-application {
+      border-left: 5px solid $darkerGrayBlueColor;
```

Comment:
should probably add a transparent border to all the other apps to align things properly, and `em` would be a better unit in this case (something like `0.25em`).